guiEditor/src/components/parentingPropertyGridComponent.tsx

```diff
@@ -1,75 +1,50 @@
 import * as React from "react";
-import { OptionsLineComponent } from "../sharedUiComponents/lines/optionsLineComponent";
-import { LineContainerComponent } from "../sharedUiComponents/lines/lineContainerComponent";
-import { GlobalState } from "../globalState";
 import { Control } from "babylonjs-gui/2D/controls/control";
-import { Container } from "babylonjs-gui/2D/controls/container";
+import { TextLineComponent } from "../sharedUiComponents/lines/textLineComponent";
+import { FloatLineComponent } from "../sharedUiComponents/lines/floatLineComponent";
+import { LockObject } from "../sharedUiComponents/tabs/propertyGrids/lockObject";
+import { PropertyChangedEvent } from "../sharedUiComponents/propertyChangedEvent";
+import { Observable } from "babylonjs/Misc/observable";
+import { Grid } from "babylonjs-gui/2D/controls/grid";
 
 interface IParentingPropertyGridComponentProps {
-    guiNode: Control;
-    guiNodes: Control[];
-    globalState: GlobalState;
+    control: Control,
+    lockObject: LockObject,
+    onPropertyChangedObservable?: Observable<PropertyChangedEvent>
 }
 
 export class ParentingPropertyGridComponent extends React.Component<IParentingPropertyGridComponentProps> {
     constructor(props: IParentingPropertyGridComponentProps) {
         super(props);
+        const cellInfo = (this.props.control.parent as Grid).getChildCellInfo(this.props.control);
+        this._rowNumber = parseInt(cellInfo.substring(0, cellInfo.search(":")));

```

Comment:
On certain cases this will be NaN (next line as well). Would be great to stay defensive and make sure it defaults to a value during construction.